src/GitTabExpansion.ps1

```diff
@@ -330,6 +351,21 @@ function GitTabExpansionInternal($lastBlock) {
             gitBranches $matches['ref'] $true
             gitTags $matches['ref']
         }
+
+        # Handles git <cmd> --<param>=<value>
+        "^(?<cmd>$($someCommands -join '|')).* --(?<param>[^=]+)=(?<value>\S*)$" {

```

Comment:
I'm not _terribly_ concerned about performance, but can we precompute `$someCommands -join '|'` since it's used a couple times?

More generally, It seems a little curious to use `$someCommands` here rather than explicitly looking for the keys in `$paramvalues` (or `$params`/`$shortparams`). Does everything still work as expected if a command is found in `$someCommands` but not one of the params lists? If not, we might consider a test to iterate over `$someCommands` (via tab-completing `git <tab>`) checking if long and short parameters can be completed successfully.